payloads/v10/user.ts

```diff
@@ -348,3 +354,56 @@ export interface APIAvatarDecorationData {
 	 */
 	sku_id: Snowflake;
 }
+
+/**
+ * The collectibles the user has, excluding Avatar Decorations and Profile Effects.
+ *
+ * @see {@link https://discord.com/developers/docs/resources/user#collectibles}
+ */
+export interface APICollectibles {
+	/**
+	 * Object mapping of {@link APINameplateData}
+	 */
+	nameplate?: APINameplateData;
+}
+
+/**
+ * @see {@link https://discord.com/developers/docs/resources/user#nameplate}
+ */
+export interface APINameplateData {
+	/**
+	 * ID of the nameplate SKU
+	 */
+	sku_id: Snowflake;
+	/**
+	 * Path to the nameplate asset
+	 *
+	 * @example `nameplates/nameplates/twilight/`
```

Comment:
Lets have a CDN route function thing for nameplates 👁️

Reply:
I didn't add it because its not documented and because the structure of it is not exactly clear.
From my testing, nameplates can be either prefixed with `nameplates/nameplates/` or `nameplates/nameplates_v2/`..

Reply:
Eugh, they really did nameplates dirty huh.. well w/e